### PR TITLE
#4719[FEATURE REQUEST] Integration with CalDav for Jitsi meetings

### PR DESCRIPTION
**Pull request by Abhishek260305**
I have made an outline for the integration of the CalDav with calendars of Jitsi, It is not complete yet. Please review it and complete it, most of it is done, please help in completing it.

### CODE REVIEW

File: react/features/calendar-sync/web/CalDav.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { IStore } from '../../app/types';
+import {
+  getCalendarEntries,
+  loadCalDavAPI,
+  signIn,
+  updateCalendarEvent,
+  updateProfile
+} from '../../caldav-api/actions'; // @ts-ignore
+import calDavApi from '../../caldav-api/calDavApi.web';
```
Comment (damencho) [inline react/features/calendar-sync/web/CalDav.ts:9]: Did you forget to include this in the PR?

**Review by damencho — CHANGES_REQUESTED** [review]

File: react/features/calendar-sync/web/CalDav.ts (hunk 1 above)
Comment (Abhishek260305) [reply react/features/calendar-sync/web/CalDav.ts:9]: Respected Sir,
I am having trouble in completing "from '../../caldav-api/actions'; // @ts-ignore
import calDavApi from '../../caldav-api/calDavApi.web';"
Kindly help me through this.

Comment (damencho) [reply react/features/calendar-sync/web/CalDav.ts:9]: The reason that this was not implemented long time ago is missing a good js library  that implements caldav. Implementing and supporting a library like that is a lot of work and should happen outside of jitsi-meet.

Comment (Abhishek260305) [reply react/features/calendar-sync/web/CalDav.ts:9]: So is there any other possible way to do it?
I want to work on it and complete it.

Comment (damencho) [reply react/features/calendar-sync/web/CalDav.ts:9]: Start simple. Find a library or implement something that is being able to communicate  caldav outside jitsi-meet. A simple example. When you have it then try to integrate it in jitsi-meet.

**Comment by damencho** [comment]
I will close this for now.

**Pull request closed without merging**